--- main/mainworker.cpp	
+++ main/mainworker.cpp	
@@ -833,9 +833,9 @@
 		std::string sbackup_DirM = backup_DirM.str();
 
 		//create folders if they not exists
-		mkdir_deep(sbackup_DirH.c_str(),777);
-		mkdir_deep(sbackup_DirD.c_str(),777);
-		mkdir_deep(sbackup_DirM.c_str(),777);
+		mkdir_deep(sbackup_DirH.c_str(),0755);
+		mkdir_deep(sbackup_DirD.c_str(),0755);
+		mkdir_deep(sbackup_DirM.c_str(),0755);
     	
 		time_t now = mytime(NULL);
 		struct tm tm1;
